### ISSUE TITLE
Progress bar naming

### ISSUE THREAD

**Issue by marcchataigner** [dat-land/dat-desktop#609]
Progress bar naming -- Linked with the #597 DAT Desktop v4 concept

The status is as much about 'download' for the first time you fetch the Dat (in that case, 0-->100% makes sense), as about 'upload' (then the file is 'full', but '100%' says something else like 'complete' downloaded), as about 'updating' (when the Dat-folder will keep itself 'synced').

Labeling of these 3 difference states can still be renamed differently; any proposition ?

<img width="424" alt="screen shot 2018-11-29 at 11 00 59" src="https://user-images.githubusercontent.com/3974900/49194365-284cdc00-f3c6-11e8-8935-2764b73fd9fc.png">

**Comment by dkastl** [comment]
Maybe the colors could tell something about the synchronization status:

* `green` -> available/normal/healthy
* `blue` -> updating/syncing
* `red` -> error

And the progress bar indicator would tell something about the completeness of the local node:

* `100%` -> complete local node
* < 100%` -> partially available on local node

For example, would it be possible to have a "70% red" or "70% green"? What does error mean, if the progress is 100%? How would you show  a color, when the progress is 0%?

**Comment by marcchataigner** [comment]
> (...) would it be possible to have a "70% red" or "70% green"? What does error mean, if the progress is 100%? How would you show a color, when the progress is 0%?

Good point; the back ground color is light grey, so when 0%, only light grey might be visible. And then blue as the download/sync goes on.

Error needs to be explained indeed.

**Comment by AtuyL** [comment]
I think ... Progress is required bar-style ?
We can visible the percentage.
It is example. this ux is use only thin spaces.

![screenshot 2018-11-30 13 51 06](https://user-images.githubusercontent.com/176803/49269204-1abb5300-f4a7-11e8-8342-7c7c5efbb5a7.jpg)

**Comment by AtuyL** [comment]
and more example.
![screenshot 2018-11-30 14 02 21](https://user-images.githubusercontent.com/176803/49269486-92d64880-f4a8-11e8-9e4a-1dfd46a03766.jpg)
